pkg/kubelet/kubelet.go

```diff
@@ -208,6 +211,14 @@ func NewMainKubelet(
 	if err = klet.volumePluginMgr.InitPlugins(volumePlugins, &volumeHost{klet}); err != nil {
 		return nil, err
 	}
+	netPluginMgr := &network.NetworkPluginMgr{}
+	if err = netPluginMgr.ValidateAndInitNetworkPlugin(networkPlugins, networkPluginName, &networkHost{klet}); err != nil {
```

Comment:
for consistency, maybe:

```
if plug, err := netPluginMgr.InitPlugin(networkPlugins, networkPluginName, &networkHost{klet}); err != nil {
    return nil, err
} else {
    klet.networkPlugin = plug
}
```

I don't see any reason to specifically call out validation here or to break it into two steps.